CaloReco/src/CaloTemplateWFProcessor.cc

```diff
@@ -258,9 +258,9 @@ namespace mu2e {
 	for (unsigned j=0;j<xvec.size();++j) ywork[j] -= fmutil_.eval_fcn(xvec[j]);	
```

Comment:
ywork is sized by yvec and filled to the limits of xvec.  I would guess that xvec and yvec are always supposed to be the same size, so it doesn't matter, but it is a little harder to follow, and the sane-size requirement is not noted anywhere. Would you mind putting in a check that they are the same size, or at least a note?

Reply:
Yes, I added a check at the beginning of the function to be sure the rest of the cod works as intended.

I also fixed tabs with spaces (my old, beloved nedit auto-tabbed this file a while ago and I fixed it)